src/gen_smtp_server_session.erl

```diff
@@ -342,10 +326,14 @@ handle_request({<<"HELO">>, <<>>}, State) ->
 handle_request({<<"HELO">>, Hostname},
 			   #state{options = Options, module = Module, callbackstate = OldCallbackState} = State) ->
 	case Module:handle_HELO(Hostname, OldCallbackState) of
+		{ok, 0, CallbackState} ->
```

Comment:
Also, this typespec needs to be updated now:

https://github.com/gen-smtp/gen_smtp/blob/628863f0bfd4439678bf216da5c1f55626ec2a8e/src/gen_smtp_server_session.erl#L90-L91

because it says `pos_integer()`